cidc_api/models/files/facets.py

```diff
@@ -197,9 +197,8 @@ def __init__(self, match_clauses: List[str], description: Optional[str] = None):
         )
     },
     "IHC": {
-        "All IHC Files": FacetConfig(
-            ["Assay Type|IHC|All IHC Files|/ihc", "/ihc/ihc_image."]
-        )
+        "Images": FacetConfig(["/ihc/ihc_image."]),
```

Comment:
dot at the end feels weird, but what do i know

Reply:
That's the auto-generated facet group from prism.